Jenkinsfile

```diff
@@ -37,25 +34,65 @@ pipeline{
 		stage('Main: Run Data-Export'){
 			steps{
 				script{
-					sh "mkdir -p ${folder}"
+					def releaseVersion = utils.getReleaseVersion()
+					sh "mkdir -p ${env.OUTPUT_FOLDER}"
 					withCredentials([usernamePassword(credentialsId: 'neo4jUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-						sh "java -Xmx${env.JAVA_MEM_MAX}m -jar target/data-export-jar-with-dependencies.jar --user $user --password $pass --output ./${folder} --verbose"
+						// This is a very memory-intensive step, and as such it is necessary to stop unused services to get it to run to completion.
+						// At time of writing, the source of the problem seems to come from the 'Ensembl' mappings.
```

Comment:
Could you date this comment after the phrase "At time of writing" so we have a record of when this issue was found?

Reply:
Oh, good catch, I'll do that now